cluster-autoscaler/cloudprovider/azure/README.md

```diff
@@ -68,6 +68,12 @@ or to autoscale multiple VM scale sets:
         - --nodes=1:10:k8s-nodepool-2-vmss
 ```
 
+To allow scaling similar node pools simultaneously, or when using separate node groups per zone and to keep nodes balanced across zones, use the `--balance-similar-node-groups` flag. Add it to the `command` section to enable it:
+
+```yaml
+        - --balance-similar-node-groups=true
```

Comment:
@feiskyer do you know if this flag defaults to true?

Reply:
the default value is false.